### PR TITLE
Add examples for the package

### LINKED ISSUE #6721: Add example usecase to the `ballerinax/stripe` package
**Description:**
$subject

### PR DESCRIPTION
**Pull request by ayeshLK**
Fixes: [#6721](https://github.com/ballerina-platform/ballerina-library/issues/6721)
Part of: [#6775](https://github.com/ballerina-platform/ballerina-library/issues/6775)

## Examples
N/A

## Checklist
- [X] Linked to an issue
- [ ] ~Updated the changelog~
- [ ] ~Added tests~
- [ ] ~Updated the spec~
- [ ] ~Checked native-image compatibility~

### CODE REVIEW

File: examples/manage-one-time-charges/Manage Stripe One-Time Charges.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Refer to the [Setup guide](https://central.ballerina.io/ballerinax/stripe/latest#setup-guide) to set up your stripe
+account, if you do not have one.
+
+### 2. Configuration
+
+Update your Stripe account-related configurations in the `Config.toml` file in the example root directory:
+
+```toml
+secretKey="<secret-key>"
+```
```
Comment (NipunaRanasinghe) [inline examples/manage-one-time-charges/Manage Stripe One-Time Charges.md:18]: I'm seeing inconsistencies with the indentation of this config.toml configurations (compared to L16 in `examples/README.md`). So shall we follow a single convention in both places?

File: examples/manage-one-time-charges/main.bal [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+// under the License.
+
+import ballerina/io;
+import ballerinax/stripe;
+
+// Configuration for Stripe API access
+configurable string secretKey = ?;
+
+public function main() returns error? {
+    stripe:Client stripe = check new ({auth: {token: secretKey}});
```
Comment (NipunaRanasinghe) [inline examples/manage-one-time-charges/main.bal:24]: Usually in connector examples we initialize the client in the module-level. Any particular reason to deviate from that?

File: examples/manage-payments/main.bal [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    payment = check stripe->/payment_intents/[payment.id]/confirm.post(confirmation);
+    io:println("Payment confirmed successfully: ", payment.id);
+
+    // Refund a confirmed payment
+    stripe:refunds_body refundDetails = {
+        // Update the payment-id here
+        payment_intent: payment.id
+    };
+    stripe:Refund paymentRefund = check stripe->/refunds.post(refundDetails);
+    io:println("Payment refund successful: ", paymentRefund.id);
```
Comment (NipunaRanasinghe) [inline examples/manage-payments/main.bal]:
```suggestion
    io:println("Payment refund was successful. Refund ID: ", paymentRefund.id);
```

File: examples/manage-one-time-charges/main.bal (hunk 2 above)
Comment (ayeshLK) [reply examples/manage-one-time-charges/main.bal:24]: Bit confused. AFAIR examples are kind of BBEs for the connectors and in BBEs module-level client initialization is a bad-practice. Hence didn't do that here as well. I checked  some other connector examples [1][2] as well, they hv already followed the same structure.

[1] - https://github.com/ballerina-platform/module-ballerinax-googleapis.gmail/blob/master/examples
[2] - https://github.com/ballerina-platform/module-ballerinax-zendesk/blob/main/examples

File: examples/manage-one-time-charges/Manage Stripe One-Time Charges.md (hunk 1 above)
Comment (ayeshLK) [reply examples/manage-one-time-charges/Manage Stripe One-Time Charges.md:18]: In the base Readme we have to indent code-blocks as there are few points. As an example in the `Running an Example` section we have two points, `1. To build an  example` and `2. To run an example`. Hence, it is correct to indent the code into another level. But, in the specific examples Readme we don't have that issue hence, no need to indent it much further.

Comment (NipunaRanasinghe) [reply examples/manage-one-time-charges/Manage Stripe One-Time Charges.md:18]: Ack, that makes sense

File: examples/manage-one-time-charges/main.bal (hunk 2 above)
Comment (NipunaRanasinghe) [reply examples/manage-one-time-charges/main.bal:24]: Sorry, my mistake. I confused this with the quickstart guide where we initialize the client at the module level. All good!

**Review by NipunaRanasinghe — DISMISSED** [review]

**Review by NipunaRanasinghe — APPROVED** [review]

**Pull request merged**